bezier-rs/docs/interactive-docs/src/components/SliderExample.vue

```diff
@@ -1,15 +1,17 @@
 <template>
 	<div>
-		<Example :title="title" :bezier="bezier" :callback="callback" :options="value.toString()" />
-		<div class="slider_label">{{ templateOptions.variable }} = {{ value }}</div>
-		<input class="slider" v-model="value" type="range" :step="templateOptions.step" :min="templateOptions.min" :max="templateOptions.max" />
+		<Example :title="title" :bezier="bezier" :callback="callback" :options="$data" />
+		<div v-for="(slider, index) in templateOptions.sliders" :key="index">
+			<div class="slider_label">{{ slider.variable }} = {{ $data[slider.variable] }}</div>
+			<input class="slider" v-model.number="$data[slider.variable]" type="range" :step="slider.step" :min="slider.min" :max="slider.max" />
```

Comment:
I'd feel more comfortable if this `$data[slider.variable]` was done in a method or computed function.

Reply:
We could create a function here using v-on="() => {update(variable.name, ...)}" but this is what v-model does under the hood already. Using an function here is also not ideal as the goal in this component is to allow the slider's variable names to be dynamic, which v-model handles very succinctly. I think the use of $data here though is unclean and we shouldn't be passing it directly to the child component, so instead I've created a sub-object within data to be modified and passed instead.